### DISCUSSION TITLE
inception_bak 授权问题

### DISCUSSION THREAD

**Discussion by chenjinlei1122** [hhyo/Archery#2547] [Q&A]
inception_bak是需要在inception所在的mysql实例进行GRANT SELECT, INSERT, CREATE ON *.* TO 'inception_bak'@'archery所在主机ip'操作，还是要在每一个实例都进行grant 授权操作

**Comment by hhyo** [comment]
inception所在机器和archery所在机器，前者用于inception生成备份，后者用于平台页面获取回滚语句

**Comment by chenjinlei1122** [comment]
archery, inception 在同个机器，用的同个数据库实例，是不是只要在这个数据库实例进行grant操作就可以了
